CHANGELOG.rst

```diff
@@ -2,6 +2,10 @@
 
  - Remove explicit support for Python 3.8
 
+**2.0.2 - 10/05/23**
+
+ - Minor bugfix to ensure default remission rate calls the right artifact key
```

Comment:
Did we actually release VPH 2.0.2? I didn't think we did. If we didn't, this information should be additional bullet point in the 2.1.0 release.